docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -400,22 +405,25 @@ quarkus.oidc.authentication.error-path=/error
 ----
 
 Ensure that the property starts with a forward slash (/) character and the path is relative to the base URI of the current endpoint.
-For example, if it is set as '/error' and the current request URI is `https://localhost:8080/callback?error=invalid_scope` then a final redirect will be made to `https://localhost:8080/error?error=invalid_scope`.
+For example, if it is set to '/error' and the current request URI is `https://localhost:8080/callback?error=invalid_scope`, then a final redirect is made to `https://localhost:8080/error?error=invalid_scope`.
 
-It is important to ensure that this error endpoint is a public resource to prevent the user from being redirected to this page to be authenticated again.
+[IMPORTANT]
+====
+To prevent the user from being redirected to this page to be reauthenticated, ensure that this error endpoint is a public resource.
+====
 
 === Accessing authorization data
 
-Let's first discuss how to access information around authorization.
+You can access information around authorization in different ways.
 
 [[access_id_and_access_tokens]]
 ==== Accessing ID and access tokens
 //SJ: new concept topic to describe the different token types and usage proposed in next iteration
```

Comment:
```suggestion
//SJ: New concept topic to describe the different token types and usage proposed in the next iteration.
```

Reply:
Just a comment that I added to this topic. Shall leave this as is.